rasa/core/events/__init__.py

```diff
@@ -1167,6 +1166,23 @@ def apply_to(self, tracker: "DialogueStateTracker") -> None:
         tracker.change_form_to(self.name)
```

Comment:
should that be `change_active_loop_to` now?

Reply:
great catch!